components/servo/webview.rs

```diff
@@ -445,4 +446,17 @@ impl WebView {
     pub fn paint(&self) -> bool {
         self.inner().compositor.borrow_mut().render()
     }
+
+    pub fn execute_js(&self, script: String, sender: IpcSender<WebDriverJSResult>) {
+        let bcid: BrowsingContextId = self.id().into();
+
+        self.inner()
+            .constellation_proxy
+            .send(ConstellationMsg::WebDriverCommand(
+                WebDriverCommandMsg::ScriptCommand(
+                    bcid,
+                    WebDriverScriptCommand::ExecuteScript(script, sender),
+                ),
+            ));
+    }
```

Comment:
We are not going to be exposing `IpcSender` to the API. Let's have a conversation about how this API should look.

Reply:
Ok what would be a good way to do this then? I am not familiar yet with what should and shouldn't be done in servo but I didn't want to block in the webview impl.